tests/Integration/UseCases/CancelDonation/CancelDonationUseCaseTest.php

```diff
@@ -134,7 +144,8 @@ private function newUseCaseWithMailerMock( Donation $donation ): CancelDonationU
 
 		return new CancelDonationUseCase(
 			$this->getDonationRepositoryWithDonation( $donation ),
-			$mailer
+			$mailer,
+			new SucceedingAuthorizer()
```

Comment:
The new tests I wrote for the new functionality are in the route. Modifications in this file are to keep these tests working as they did before.